### PR TITLE
global: general refactor

### PR DESCRIPTION
**Pull request by egabancho**

### CODE REVIEW

File: cds_dojson/matcher.py [hunk 1]
```diff
@@ -42,3 +46,45 @@ def query(self):
...
+
+
+def matcher(record, entry_point):
+    """DoJSON model matcher.
+
+    :param record: Something that looks like a python dictionary
+
+    :returns: a model instance
+    """
+    logger = logging.getLogger(__name__ + ".dojson_matcher")
```
Comment (hachreak) [inline cds_dojson/matcher.py]: you can rename as `__name__ + ".matcher"`

Comment (egabancho) [inline cds_dojson/matcher.py]: matcher seemed to be to generic ...

File: cds_dojson/matcher.py [hunk 2]
```diff
@@ -42,3 +46,45 @@ def query(self):
     def match(self, record, user_info=None):
         """Return True if record match the query."""
         return self.query.accept(MatchUnit(record))
+
+
+def matcher(record, entry_point):
```
Comment (hachreak) [inline cds_dojson/matcher.py]: maybe you can rename `entry_point` with `entry_point_group`.
It should be more clear (and there is no ambiguity with the `entry_point` inside the `foreach`).

Comment (egabancho) [inline cds_dojson/matcher.py]: :+1:

**Pull request closed without merging**